### PR TITLE
fix broken links in readme files

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by lhein** [comment]
Right. I will have a look later

**Comment by lhein** [comment]
I tried to make the paths relative but that is not working. Github maps content of the .github folder like Readme.md into the root of the repository. So if I have relative links in that file this will work when clicking the readme links inside the .github folder but it will fail when clicking in the root of the repository. 
I decided to stay with the full links as that seems the only way having it work always.

**Comment by apupier** [comment]
Reconsidering this move to a .github folder. In fact, the Readme, contributing.md, build.md, Code of conduct  are not really specific to github, it is a "normal" project information. The only specific files to github are the PR template, the issue template and the Codeowner.

File: Contributing.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The general idea is to keep your 'master' branch in-sync with the 'upstream/mast
 
 ## Building JBoss Fuse Tooling
 
-To build _JBoss Fuse Tooling_ requires specific versions of Java (1.8+) and +Maven (3.0+). See this [link](https://github.com/jbosstools/jbosstools-fuse/blob/master/Build.md) for more information on how to setup, run and configure build.
+To build _JBoss Fuse Tooling_ requires specific versions of Java (1.8+) and +Maven (3.0+). See this [link](./Build.md) for more information on how to setup, run and configure build.
```
Comment (apupier) [inline Contributing.md]: in the example they are not using a "./Build.md" but directly "Build.md".
Do you know if there is a difference?

Comment (lhein) [reply Contributing.md]: no idea. I did not look at the examples. I tested the link with ./filename and that seems to work fine.

**Pull request merged**